pilot/docker/Dockerfile.proxyv2

```diff
@@ -43,7 +52,6 @@ COPY pilot-agent /usr/local/bin/pilot-agent
 COPY envoy_pilot.yaml.tmpl /etc/istio/proxy/envoy_pilot.yaml.tmpl
 COPY envoy_policy.yaml.tmpl /etc/istio/proxy/envoy_policy.yaml.tmpl
 COPY envoy_telemetry.yaml.tmpl /etc/istio/proxy/envoy_telemetry.yaml.tmpl
-COPY istio-iptables.sh /usr/local/bin/istio-iptables.sh
```

Comment:
Confused - why do you move this line up ? (and again, why rename )

Reply:
In distroless we use the golang implementation, in ubuntu we use bash. If it is here then its adding the bash one to distroless